MediaBrowser.Api/Syncplay/TimeSyncService.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using MediaBrowser.Controller.Configuration;
+using MediaBrowser.Controller.Net;
+using MediaBrowser.Controller.Session;
+using MediaBrowser.Model.Services;
+using MediaBrowser.Model.Syncplay;
+using Microsoft.Extensions.Logging;
+
+namespace MediaBrowser.Api.Syncplay
+{
+    [Route("/GetUtcTime", "GET", Summary = "Get UtcTime")]
+    [Authenticated]
+    public class GetUtcTime : IReturnVoid
+    {
+        // Nothing
+    }
+
+    /// <summary>
+    /// Class TimeSyncService.
+    /// </summary>
+    public class TimeSyncService : BaseApiService
+    {
+        /// <summary>
+        /// The session manager.
+        /// </summary>
+        private readonly ISessionManager _sessionManager;
+
+        /// <summary>
+        /// The session context.
+        /// </summary>
+        private readonly ISessionContext _sessionContext;
+
+        public TimeSyncService(
+            ILogger<TimeSyncService> logger,
+            IServerConfigurationManager serverConfigurationManager,
+            IHttpResultFactory httpResultFactory,
+            ISessionManager sessionManager,
+            ISessionContext sessionContext)
+            : base(logger, serverConfigurationManager, httpResultFactory)
+        {
+            _sessionManager = sessionManager;
+            _sessionContext = sessionContext;
+        }
+
+        /// <summary>
+        /// Handles the specified request.
+        /// </summary>
+        /// <param name="request">The request.</param>
+        /// <value>The current UTC time response.</value>
+        public UtcTimeResponse Get(GetUtcTime request)
+        {
+            // Important to keep the following line at the beginning
+            var requestReceptionTime = DateTime.UtcNow.ToUniversalTime().ToString("o");
+
+            var response = new UtcTimeResponse();
+            response.RequestReceptionTime = requestReceptionTime;
+            var currentSession = GetSession(_sessionContext);
```

Comment:
`currentSession` seems unused?

Reply:
I forgot to remove it. I was testing some "computation load" between the request being received and the response being sent.